library/ssl_tls13_client.c

```diff
@@ -792,15 +792,16 @@ int ssl_create_binder( mbedtls_ssl_context *ssl, unsigned char *psk, size_t psk_
     {
         ret = mbedtls_ssl_tls1_3_derive_secret( ssl, mbedtls_md_get_type( md ),
                             ssl->handshake->early_secret, hash_length,
-                            ( const unsigned char * )"res binder", sizeof( "res binder" ),
+                            ( const unsigned char * )"res binder", strlen ( "res binder" ),
```

Comment:
@hannestschofenig Why was this change made? I remember making exactly the reverse change in places recently. The length of a constant string can and should be evaluated at compile-time, IMO.